### PR TITLE
🔩 Android channel claim

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
screencast 2019-02-13 22-09-37

File: templates/public/public_android.haml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% extends "frame.html" %}
+
+-load smartmin temba compress i18n
+
+-block nav
+
+
+-block content
```
Comment (norkans7) [inline templates/public/public_android.haml]: rapidpro - visually build interactive sms applications 2019-02-13 22-14-24

**Comment by norkans7** [comment]
Just to compare screenshots without markers

screencast 2019-02-13 22-02-56

**Comment by nicpottier** [comment]
Do they need to enable things BEFORE the download? I thought it prompted you to go to the settings app when you downloaded it / clicked on it?

**Comment by norkans7** [comment]
Correct, I change to use that

**Comment by norkans7** [comment]
screencast 2019-02-14 18-28-27

File: temba/apks/views.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+from django.http import HttpResponseRedirect
+
+from .models import Apk
+
+
+class ApkCRUDL(SmartCRUDL):
+    model = Apk
+    permissions = True
+    actions = ("create", "read", "update", "list", "download")
```
Comment (dodobas) [inline temba/apks/views.py]: shouln't we add permissions in `settings_common.py` for these actions?

Comment (norkans7) [reply temba/apks/views.py]: Sure

**Comment by nicpottier** [comment]
Nice, looks good. Default images should be RapidPro and we should have an easy way of changing those to TextIt images for that install. (guessing we overload that template in TextIt?)

**Comment by nicpottier** [comment]
Couple other things:
  * can we make all these images the same height? Let's cut out the middle the image with the "install" button on it to bring it into the same range.
 * I know we had it before but I'm not sure we need all the highlighting honestly
 * What is the download link? Thought we were going to have a shorter link for them? Wouldn't we want them to just type in a URL on their phone instead of clicking a button that requires them to log into their account on a relayer? (which we probably don't want from a security standpoint)

**Comment by ericnewcomer** [comment]
I don't think we need a download page, just a link to download the latest APK. That should be a publicly available link (like, https://textit.in/androidchannel or something else short and sweet). Then we stick a button for that link on the android claim page. The APK should be in charge of fetching the message packs once it is installed based on which ones they have installed already (this already works today, it takes them to the appropriate pack, but in the play store).

**Comment by norkans7** [comment]
@ericnewcomer the download page is publicly available on https://textit.in/android 
I thought that page could be useful to show more details so a use can know if they need to install an update and the packs are there since I thought some where being remove from the store too

**Comment by norkans7** [comment]
And the latest Android channel claim page 
screencast 2019-02-15 16-07-34

File: templates/public/public_android.haml (hunk 1 above)
Comment (norkans7) [reply templates/public/public_android.haml]: rapidpro - visually build interactive sms applications 2019-02-18 13-35-56

a few updates

**Comment by nicpottier** [comment]
Tweaked this a bit more, @rowanseymour can you take one last look before we ship?

**Pull request merged**